haskell/repl.bzl

```diff
@@ -463,7 +482,7 @@ def _create_repl(hs, posix, ctx, repl_info, output):
                 hs.tools.ghci,
                 ghci_repl_script,
             ],
-            transitive_files = inputs,
+            transitive_files = depset(transitive = [inputs, repl_info.java_deps]),
```

Comment:
I would have thought the runfiles of the haskell library should contain already the jars, but in my tests it doesn't. So I added them here. What am I missing?

Reply:
I thought that was the case. But after the latest edits it seems I don't need it anymore. I might have tested it wrong before. I just removed it.